internal/controller/testutils/agent.go

```diff
@@ -361,9 +361,11 @@ func NewDatadogAgentWithGlobalConfigSettings(namespace string, name string) v2al
 				AppKey: apiutils.NewStringPointer("my-app-key"),
 			},
 		},
-		Registry:                   apiutils.NewStringPointer("my-custom-registry"),
-		LogLevel:                   apiutils.NewStringPointer("INFO"),
-		Tags:                       []string{"tagA:valA", "tagB:valB"},
+		Registry: apiutils.NewStringPointer("my-custom-registry"),
+		LogLevel: apiutils.NewStringPointer("INFO"),
+		Tags:     []string{"tagA:valA", "tagB:valB"},
+		Env: []v1.EnvVar{{Name: "some-envA", Value: "some-valA"},
```

Comment:
```suggestion
		Env: []v1.EnvVar{
			{
				Name:  "some-envA",
				Value: "some-valA",
			},
			{
				Name:  "some-envB",
				Value: "some-valB",
			},
		},
```

Changing format to keep in line with the current format in this file (refer to function `NewDatadogAgentWithOverrides`) already and for clarity